README.md

```diff
@@ -1 +1,24 @@
-# jsonforms-vue-seed
\ No newline at end of file
+# jsonforms-vue-seed
+
```

Comment:
Please add a small description similar to the [React seed](https://github.com/eclipsesource/jsonforms-react-seed). The rest of the Readme can be removed. Also add a link to the JSON Forms [Vue docu](https://jsonforms.io/docs/integrations/vue)